extensions/jackson/src/test/groovy/io/jsonwebtoken/jackson/io/JacksonDeserializerTest.groovy

```diff
@@ -136,6 +137,6 @@ class JacksonDeserializerTest {
     }
 
     private String base64(String input) {
-        return Base64.encoder.encodeToString(input.getBytes('UTF-8'))
+        return new Base64Encoder().encode(input.getBytes('UTF-8'))
```

Comment:
Is there a reason you didn't use `Encoders.BASE64` to avoid instantiation?

Reply:
Ah, I see - it's just a test.  Still...

Reply:
I’ll fix that, originally I was using Java.util.base64, but that failed on Java7 :(